plugins/statusscreen/statusscreen.go

```diff
@@ -49,7 +49,7 @@ func run(plugin *node.Plugin) {
 	content.SetColumns(0)
 
 	footer := newPrimitive("")
-	footer.SetBackgroundColor(tcell.ColorDarkMagenta)
+	footer.SetBackgroundColor(tcell.ColorDarkBlue)
```

Comment:
If you really want to change the colors then it should be something that anybody can change. Maybe try adding the color as a parameter

Reply:
Ok, but this is as far as my golang experience goes 😂

Reply:
I like the idea of having "themes" - we can just add a public variable for the colors and a 2nd plugin can then just change these Public colors from another plugin

So you can have stattusscreen-dark-theme or sth as a plugin and people can change to whatever they want - for the scope of this PR can you remove the color changes?